app/Resources/views/article.html.twig

```diff
@@ -316,6 +316,66 @@
 
     {%- endif %}
 
+    <script type="application/ld+json">
+        {
+            "@context": "https://schema.org",
+            "@type": "ScholarlyArticle",
+            "mainEntityOfPage": {
+                "@type": "WebPage",
+                "@id": "{{ url('article', {id: item.id}) }}"
+            },
+            "headline": "{{ item.fullTitle|striptags }}",
+            "datePublished": "{{ item.publishedDate|date('Y-m-d', 'UTC') }}",
+            "author": [
+                {% for author in item.authors %}
+                    {% if author.orcid is defined %}
+                        {
+                            "@type": "Person",
+                            "name": "{{ author.preferredName }}"
+                        }
+                    {% elseif author.people is defined %}
+                        {
+                            "@type": "Organization",
+                            "name": "{{ author.toString() }}"
+                        }
+                    {% else %}
+                        "{{ author.toString() }}"
+                    {% endif %}
+                    {%- if not loop.last %}, {% endif -%}
+                {% endfor %}
+            ],
+            "publisher": {
+                "@type": "Organization",
+                "name": "eLife Sciences Publications, Ltd",
+                "logo": "{{ absolute_url(asset('assets/patterns/img/patterns/organisms/elife-logo-full.svg')) }}"
```

Comment:
```suggestion
                "logo": {
                    "@type": "ImageObject",
                    "url": "{{ absolute_url(asset('assets/patterns/img/patterns/organisms/elife-logo-full.svg')) }}"
                }
```

As we discussed yesterday, seems that the Google validator (at least) wants the more verbose option from https://github.com/elifesciences/journal/pull/1187#discussion_r333058048.